extensions/CoreWidgetProvider/Widgets/Templates/SystemMemoryTemplate.json

```diff
@@ -1,152 +1,178 @@
-                  "size": "small",
-                  "isSubtle": true,
-                  "horizontalAlignment": "right"
-                },
-                {
-                  "text": "${allMem}",
-                  "type": "TextBlock",
-                  "size": "large",
-                  "weight": "bolder",
-                  "horizontalAlignment": "right"
-                }
-              ]
-            }
-          ]
-        },
-        {
-          "type": "ColumnSet",
-          "columns": [
-            {
-              "type": "Column",
-              "items": [
-                {
-                  "text": "%Memory_Widget_Template/Committed%",
-                  "type": "TextBlock",
-                  "size": "small",
-                  "isSubtle": true
-                },
-                {
-                  "text": "${committedMem}/${committedLimitMem}",
-                  "type": "TextBlock",
-                  "size": "medium"
-                }
-              ]
-            },
-            {
-              "type": "Column",
-              "items": [
-                {
-                  "text": "%Memory_Widget_Template/Cached%",
-                  "type": "TextBlock",
-                  "size": "small",
-                  "isSubtle": true,
-                  "horizontalAlignment": "right"
-                },
-                {
-                  "text": "${cachedMem}",
-                  "type": "TextBlock",
-                  "size": "medium",
-                  "horizontalAlignment": "right"
-                }
-              ]
-            }
-          ]
-        },
-        {
-          "type": "ColumnSet",
-          "columns": [
-            {
-              "type": "Column",
-              "items": [
-                {
-                  "text": "%Memory_Widget_Template/PagedPool%",
-                  "type": "TextBlock",
-                  "size": "small",
-                  "isSubtle": true
-                },
-                {
-                  "text": "${pagedPoolMem}",
-                  "type": "TextBlock",
-                  "size": "medium"
-                }
-              ]
-            },
-            {
-              "type": "Column",
-              "items": [
-                {
-                  "text": "%Memory_Widget_Template/NonPagedPool%",
-                  "type": "TextBlock",
-                  "size": "small",
-                  "isSubtle": true,
-                  "horizontalAlignment": "right"
-                },
-                {
-                  "text": "${nonPagedPoolMem}",
-                  "type": "TextBlock",
-                  "size": "medium",
-                  "horizontalAlignment": "right"
-                }
-              ]
-            }
-          ]
-        }
-      ]
-    }
-  ],
-  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
-  "version": "1.5"
-}
\ No newline at end of file
+Here's the corrected Adaptive Card JSON with the brackets fixed:
```

Comment:
Please don't put extra text in this file.

Reply:
resolved this change @krschau